etl-job/fhir_import.py

```diff
@@ -12,17 +13,21 @@
 def _get_token() -> str:
     """Get ACCESS_TOKEN from environment"""
     # print("[out] retrieving access token...")
-    return os.environ['ACCESS_TOKEN']
+    return os.environ.get('ACCESS_TOKEN', None)
 
 
 def _auth(access_token: str) -> Gen3Auth:
     """Authenticate using ACCESS_TOKEN"""
     # print("[out] authorizing...")
-    return Gen3Auth(refresh_file=f"accesstoken:///{access_token}")
+    if access_token:
+        # use access token from environment (set by sower)
+        return Gen3Auth(refresh_file=f"accesstoken:///{access_token}")
```

Comment:
Does Sower pass the access_token to Fence at all? Haven't seen it or the "accesstoken:///" syntax before.

Reply:
https://github.com/uc-cdis/gen3sdk-python/blob/master/gen3/auth.py#L136